server/render/index.js

```diff
@@ -17,6 +17,9 @@ import {
 	getDocumentHeadMeta,
 	getDocumentHeadLink
 } from 'state/document-head/selectors';
+import { reducer } from 'state';
+import { SERIALIZE } from 'state/action-types';
+import stateCache from 'state-cache';
 
 const debug = debugFactory( 'calypso:server-render' );
 const markupCache = new Lru( { max: 3000 } );
```

Comment:
We should give this cache a max age now that the keys are not timestamped (previously the keys only lasted for an hour, meaning entries would drop out of the LRU after that time).